lib/widgets/layout.dart

```diff
@@ -0,0 +1,66 @@
+import 'package:flutter/material.dart';
+import 'package:witnet_wallet/widgets/headerLayout.dart';
+
+class Layout extends StatelessWidget {
+  final List<Widget> widgetList;
+  final AppBar? appBar;
+  final List<Widget> actions;
+  final List<Widget> headerActions;
+  final double actionsSize;
+
+  const Layout({
+    required this.widgetList,
+    required this.actions,
+    required this.actionsSize,
+    required this.headerActions,
+    this.appBar,
+  });
+
+  Widget build(BuildContext context) {
+    final theme = Theme.of(context);
+    GlobalKey contentKey = GlobalKey();
+    return Scaffold(
+      appBar: appBar,
+      resizeToAvoidBottomInset: false,
+      backgroundColor: theme.backgroundColor,
+      body: GestureDetector(
+        onTap: () {
+          FocusScope.of(context).unfocus();
+        },
+        child: ListView(
```

Comment:
This `ListView` is breaking the app every time I use the debugger and click "Show Baselines":
```
══╡ EXCEPTION CAUGHT BY RENDERING LIBRARY ╞═════════════════════════════════════════════════════════
The following assertion was thrown during paint():
'package:flutter/src/rendering/sliver_padding.dart': Failed assertion: line 305 pos 18:
'innerRect.bottom <= outerRect.bottom': is not true.

Either the assertion indicates an error in the framework itself, or we should provide substantially
more information in this error message to help you determine and fix the underlying cause.
In either case, please report this assertion by filing a bug on GitHub:
  https://github.com/flutter/flutter/issues/new?template=2_bug.md

The relevant error-causing widget was:
  ListView ListView:file:///home/tommy/development/witnet/witnet_wallet/lib/widgets/layout.dart:30:16

When the exception was thrown, this was the stack:
#2      RenderSliverEdgeInsetsPadding.debugPaint.<anonymous closure> (package:flutter/src/rendering/sliver_padding.dart:305:18)
#3      RenderSliverEdgeInsetsPadding.debugPaint (package:flutter/src/rendering/sliver_padding.dart:310:6)
#4      RenderObject._paintWithContext.<anonymous closure> (package:flutter/src/rendering/object.dart:2803:7)
#5      RenderObject._paintWithContext (package:flutter/src/rendering/object.dart:2807:6)
#6      PaintingContext.paintChild (package:flutter/src/rendering/object.dart:239:13)
#7      RenderViewportBase._paintContents (package:flutter/src/rendering/viewport.dart:677:17)
#8      PaintingContext.pushLayer (package:flutter/src/rendering/object.dart:460:12)
#9      PaintingContext.pushClipRect (package:flutter/src/rendering/object.dart:520:7)
#10     RenderViewportBase.paint (package:flutter/src/rendering/viewport.dart:652:38)
#11     RenderObject._paintWithContext (package:flutter/src/rendering/object.dart:2796:7)
#12     PaintingContext._repaintCompositedChild (package:flutter/src/rendering/object.dart:155:11)
#13     PaintingContext.repaintCompositedChild (package:flutter/src/rendering/object.dart:98:5)
#14     PipelineOwner.flushPaint (package:flutter/src/rendering/object.dart:1116:31)
#15     RendererBinding.drawFrame (package:flutter/src/rendering/binding.dart:515:19)
#16     WidgetsBinding.drawFrame (package:flutter/src/widgets/binding.dart:884:13)
#17     RendererBinding._handlePersistentFrameCallback (package:flutter/src/rendering/binding.dart:378:5)
#18     SchedulerBinding._invokeFrameCallback (package:flutter/src/scheduler/binding.dart:1175:15)
#19     SchedulerBinding.handleDrawFrame (package:flutter/src/scheduler/binding.dart:1104:9)
#20     SchedulerBinding._handleDrawFrame (package:flutter/src/scheduler/binding.dart:1015:5)
#21     _invoke (dart:ui/hooks.dart:148:13)
#22     PlatformDispatcher._drawFrame (dart:ui/platform_dispatcher.dart:318:5)
#23     _drawFrame (dart:ui/hooks.dart:115:31)
(elided 2 frames from class _AssertionError)

The following RenderObject was being processed when the exception was fired: RenderViewport#ed87a:
  needs compositing
  creator: Viewport ← IgnorePointer-[GlobalKey#22239] ← Semantics ← Listener ← _GestureSemantics ←
    RawGestureDetector-[LabeledGlobalKey<RawGestureDetectorState>#cbfad] ← Listener ← _ScrollableScope
    ← _ScrollSemantics-[GlobalKey#14285] ← NotificationListener<ScrollMetricsNotification> ←
    RepaintBoundary ← CustomPaint ← ⋯
  parentData: <none> (can use size)
  constraints: BoxConstraints(0.0<=w<=411.4, 0.0<=h<=868.6)
  layer: OffsetLayer#15cb4
  size: Size(411.4, 868.6)
  axisDirection: down
  crossAxisDirection: right
  offset: ScrollPositionWithSingleContext#7dd32(offset: 0.0, range: 0.0..229.9, viewport: 868.6,
    ScrollableState, AlwaysScrollableScrollPhysics -> ClampingScrollPhysics ->
    RangeMaintainingScrollPhysics, IdleScrollActivity#9c697, ScrollDirection.idle)
  anchor: 0.0
This RenderObject had the following descendants (showing up to depth 5):
    center child: RenderSliverPadding#7733c relayoutBoundary=up1
      child: RenderSliverList#cabeb relayoutBoundary=up2
        child with index 0: RenderIndexedSemantics#530ad relayoutBoundary=up3
          child: RenderRepaintBoundary#4e3e6 relayoutBoundary=up4
            child: RenderClipPath#6dcfa relayoutBoundary=up5
        child with index 1: RenderIndexedSemantics#8704b relayoutBoundary=up3
          child: RenderRepaintBoundary#db726 relayoutBoundary=up4
            child: RenderPositionedBox#94424 relayoutBoundary=up5
        child with index 2: RenderIndexedSemantics#5a0ee relayoutBoundary=up3
          child: RenderRepaintBoundary#0cc26 relayoutBoundary=up4
            child: RenderConstrainedBox#c3101 relayoutBoundary=up5
════════════════════════════════════════════════════════════════════════════════════════════════════

Another exception was thrown: 'package:flutter/src/rendering/layer.dart': Failed assertion: line 813 pos 12: 'picture != null': is not true.

Another exception was thrown: Tried to paint a RenderObject reentrantly.
```

I need to look for more info because @gabaldon doesn't have this issue